services/qs-us/src/main/resources/application-sat.properties

```diff
@@ -1,13 +1,10 @@
 #Properties to update by user
+#Values can be Keys,IAM,Profile
+qs.aws.access.method=Profile
```

Comment:
Keep the values as upper case and should be same as defined as constants or enum values in java